electricitymap/contrib/capacity_parsers/OPENNEM.py

```diff
@@ -0,0 +1,135 @@
+from datetime import datetime
+
+import pandas as pd
+from requests import Response, Session
+
+from electricitymap.contrib.config import ZoneKey
+from parsers.OPENNEM import SOURCE, ZONE_KEY_TO_REGION
+
+"""Disclaimer: only works for real-time data. There is retired capacity included but we do not have the information on when the capacity was retired."""
+
+REGION_MAPPING = {
+    ZONE_KEY_TO_REGION[key]: key for key in ZONE_KEY_TO_REGION
+}  # NT only has solar capacity so it will be excluded
+
+FUEL_MAPPING = {
+    "wind": "wind",
+    "solar_rooftop": "solar",
+    "battery_charging": "battery storage",
+    "solar_utility": "solar",
+    "coal_black": "coal",
+    "battery_discharging": "battery storage",
+    "pumps": "hydro storage",
+    "gas_steam": "gas",
+    "gas_ocgt": "gas",
+    "hydro": "hydro",
+    "coal_brown": "coal",
+    "distillate": "oil",
+    "bioenergy_biogas": "biomass",
+    "gas_ccgt": "gas",
+    "gas_wcmg": "gas",
+    "gas_recip": "gas",
+    "bioenergy_biomass": "biomass",
+}
+
+
+def get_opennem_capacity_data() -> dict:
+    url = "https://api.opennem.org.au/facility/"
+    r: Response = Session().get(url)
```

Comment:
Should we do like for other parsers are allow passing the session as an arg? 
If we only intend to run things manually, I guess it would be fine to leave it as is. 
I'm asking a question in #engineering around the reason for passing sessions as args in the production parsers.

Reply:
https://electricitymaps.slack.com/archives/C01PWQYKX2R/p1698752273827549

Reply:
In the way i had planned this, we should run this manually once or twice a year so this is why i did it like that. If you think it's better the other way we can switch but it may be overkill?

Reply:
Thanks for updating, let's follow the same logic as in the other parsers, i.e providing at the higher level (fetch_production_capacity) session as an optional argument with `Session()` as the default value

Reply:
I think aligning these with the other parses as much as possible makes sense, if the data sources start providing monthly updates we should probably consider automating these and run them on a schedule (not as often as the production parsers though but maybe once per week).